src/test/groovy/org/prebid/server/functional/util/privacy/TcfConsent.groovy

```diff
@@ -102,4 +101,21 @@ class TcfConsent implements ConsentString {
             this.value = value
         }
     }
+
+    enum TcfPolicyVersion {
+
+        TCF_V2(2),
+        TCF_V3(4),
```

Comment:
This is actually not tcf version, tcf version stays the same(TCF v2), this is just policy version